src/main/java/core/basesyntax/Main/Main.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.Main;
```

Comment:
The package name should be in lowercase according to Java naming conventions. Consider renaming the package 'core.basesyntax.Main' to 'core.basesyntax.main'.

Reply:
you don't need a package for main
and packages should be in lowercase